doc/code/qml_drawer.rst

```diff
@@ -42,6 +42,12 @@ The following images are generated via the code:
     qml.drawer.use_style(style)
     fig, ax = qml.draw_mpl(circuit)(1.2345, 1.2345)
 
+Note that the shown ``pennylane`` style can be replicated exactly by having the
+Quicksand Bold font installed and refreshing the Matplotlib font cache (by
+deleting the ``fontlist`` file in the Matplotlib cache directory) — to find the
+cache directory, run ``matplotlib.get_cachedir()``. If this font is not
+available, the drawer will fall back on a default font.
```

Comment:
Could this go in the usage details section of `draw_mpl`? I'm wondering whether users have asked for this info?

Reply:
Hm, I'd rather keep it here, because that's where the visuals of all the styles are shown, so a potential discrepancy is obvious. But if you think both might be beneficial, please just tell me. :)
And yep, there was some confusion about how to get the right font to 'trigger' as expected. I remember 3–4 different people asking about it, including Korbinian.